spec/services/remove_status_service_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe RemoveStatusService do
+  subject { RemoveStatusService.new }
+
+  let!(:alice)  { Fabricate(:account, username: 'alice') }
```

Comment:
That is true. But we have already introduced Faker as well. We should choose to fix usernames with those strings or use Faker, depending on the situations.
In this case I think Faker is better for `alice` and `jeff`, and a fixed username is better for `bob`.